numpy/lib/info.py

```diff
@@ -147,8 +147,8 @@
 setxor1d         Set exclusive-or of 1D arrays with unique elements.
 in1d             Test whether elements in a 1D array are also present in
                  another array.
-isin             Test whether elements in an array are also present in
-                 another array, preserving shape of first array.
+isin             Test whether each element of one (possibly multidimensional) 
```

Comment:
Why just possibly? I think you can put `ND` before both of these arrays, to make it explicit compared to its surrounding `1D` functions